package.json

```diff
@@ -50,7 +50,7 @@
     "react-dates": "^12.3.0",
     "react-docgen": "^2.20.1",
     "react-dropzone": "^4.1.2",
-    "react-markdown": "^2.4.5",
+    "react-markdown": "^4.0",
```

Comment:
Could you remove this line in favour of doing it in a separate PR, possibly #394?

Reply:
Whoops, I got my branches mixed up apparently. This'll be reverted in the next commit.